dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/DdMappingFileUploadTask.kt

```diff
@@ -98,6 +114,7 @@ open class DdMappingFileUploadTask : DefaultTask() {
         uploader.upload(
             configuration.buildUrl(),
             mappingFile,
+            if (repositories.isEmpty()) null else repositoryFile,
```

Comment:
why not just return the file as an optional from `generateRepositoryFile` function